mmv1/products/iamworkforcepool/WorkforcePool.yaml

```diff
@@ -50,19 +50,33 @@ custom_code:
 exclude_sweeper: true
 examples:
   - name: 'iam_workforce_pool_basic'
+    primary_resource_name: 'fmt.Sprintf("tf-test-example-pool%s", context["random_suffix"])'
     primary_resource_id: 'example'
+    region_override: 'global'
     vars:
       workforce_pool_id: 'example-pool'
     test_env_vars:
       org_id: 'ORG_ID'
   - name: 'iam_workforce_pool_full'
+    primary_resource_name: 'fmt.Sprintf("tf-test-example-pool%s", context["random_suffix"])'
     primary_resource_id: 'example'
+    region_override: 'global'
     vars:
       workforce_pool_id: 'example-pool'
     test_env_vars:
       org_id: 'ORG_ID'
+iam_policy:
+  parent_resource_attribute: 'workforce_pool_id'
+  method_name_separator: ':'
+  example_config_body: 'templates/terraform/iam/iam_attributes.go.tmpl'
+  allowed_iam_role: 'roles/iam.workforcePoolViewer'
+  admin_iam_role: 'roles/iam.workforcePoolAdmin'
+  iam_conditions_request_type: 'REQUEST_BODY'
```

Comment:
Workforce pools don't seem to support IAM conditions, hence the failing tests. Removing the `iam_conditions_request_type` line should fix the failing unit tests (well, "fix" them by eliminating them).